src/mscorlib/src/System/CLRConfig.cs

```diff
@@ -9,6 +9,30 @@
 
 namespace System
 {
+    // CLRConfig is mainly reading the config switch values. this is used when we cannot use the AppContext class 
+    // one example, is using the context switch in the globalization code which require to read the switch very 
+    // early even before the appdomain get initialized.
+    // In general AppContext should be used instead of CLRConfig if there is no reason prevent that.
+    internal class CLRConfig
+    {
+
+#if CORECLR
```

Comment:
Unless this code may one day be shared with CoreRT, it will never get compiled without CORECLR defined. It's defined in System.Private.CoreLib.csproj